library/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,7 @@
 <resources>
     <string name="app_name">Android SDK</string>
     <string name="notification_error_ssl_cert_invalid">Invalid SSL certificate. Do you want to continue?</string>
+    <string name="queue_it_exiting">You are about to exiting the queue and the app, are you sure?</string>
```

Comment:
But is the text correct, I don't believe the app will be exited by using the back button? (Only if it is the last activity)

Suggestion: "You are about to exit the queue, are you sure?"